test/gtest/uct/uct_test.h

```diff
@@ -503,6 +508,23 @@ class test_uct_iface_attrs : public uct_test {
     UCS_PP_FOREACH(_UCT_INSTANTIATE_TEST_CASE, _test_case, UCT_TEST_NO_SELF_TLS)
 
 
+/**
+ * Instantiate the parametrized test case for all transports excluding CUDA.
+ *
+ * @param _test_case  Test case class, derived from uct_test.
+ */
+#define UCT_INSTANTIATE_NO_CUDA_TEST_CASE(_test_case) \
```

Comment:
maybe 
UCT_INSTANTIATE_NO_GPU_TEST_CASE 
(and remove also rocm)
reason - we can assume rocm has same "traits" as cuda